src/common/film.c

```diff
@@ -280,7 +280,7 @@ int dt_film_import(const char *dirname)
 static gboolean ask_and_delete(gpointer user_data)
 {
   GList *empty_dirs = (GList *)user_data;
-  const int n_empty_dirs = g_list_length(empty_dirs);
+  const int n_empty_dirs = g_list_is_singleton(empty_dirs) ? 1 : 2; // we only need a number for singular/plural
```

Comment:
I kinda don't agree with singular/plural but it depends on usage/wording. Let me give you an example I had in my day job:

let's say `n_dom` = number of domains
the text is "We found %d domains available".

pluralization for english is simple:
`n_dom == 1`: "We found 1 domain available";
`n_dom == 0 || n_dom > 1`: "We found 2 domains available";

For polish:
`n_dom == 0`: "Nie znaleźliśmy dostępnych domen"
`n_dom == 1`: "Znaleźliśmy 1 dostępną domenę"
`n_dom > 1 && n_dom < 5`: "Znaleźliśmy 3 dostępne domeny"
`n_dom > 4 && n_dom < 21`: "Znaleźliśmy 17 dostępnych domen"

and so on ;)

Overal for plurals and translations it's better to err on the side of full count :)

Reply:
I know about more complex plurals than in English, but didn't realize that ngettext does multiple different translations.
(Actually, even English still has a remnant of a more complex pluralization -- both/either/neither.)